src/components/General/DateAsWords.vue

```diff
@@ -1,23 +1,25 @@
 <template>
   <div>
-    {{dateInWords(date)}}
-    <q-tooltip>{{ $d(new Date(date), 'long') }}</q-tooltip>
+    {{ dateInWords }}<q-tooltip>{{ tooltipContent }}</q-tooltip>
```

Comment:
what happened to the newline?

Reply:
The CI failure seems related

Reply:
It is not related, they are "contains" matching. The test failure is because the way of faking the now date no longer works (actually it was a bit broken before as it never reset itself). I'm working on fixing up the tests, and adding some more now :)